src/mkdocstrings_handlers/python/handler.py

```diff
@@ -183,10 +183,11 @@ def load_inventory(
         Yields:
             Tuples of (item identifier, item URL).
         """
+        domains = kwargs.get("domains", ("py",)).split(",")
```

Comment:
We can definitely add this parameter to the signature, as an optional list with `None` as default value, transformed into `["py"]` if falsy:

```python
# signature...
    domains: list | None = None,
    **kwargs: Any,
# ...
```

```suggestion
        domains = domains or ["py"]
```

This will allow to take advantage of YAML types to configure it like this:

```yaml
plugins:
- mkdocstrings:
    handlers:
      python:
        import:
        - url: ...
          domains: [py, std]
```